migrations/v6-MIGRATION.md

```diff
@@ -72,6 +72,16 @@ StoreProduct has been made an interface, which `GoogleStoreProduct` and `AmazonS
 
 ### Purchasing APIs
 
+The `purchasePackage()` and `purchaseProduct()` APIs have a new behavior for selecting which offer is used when
+purchasing a `Package` or `StoreProduct`. These functions use the following logic to choose
+a [SubscriptionOption] to purchase:
+*   - Filters out offers with "rc_ignore_default_offer" tag
```

Comment:
filters out SubscriptionOptions***

hmm which makes me think, did we test a scenario where the base plan has that tag?